SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialPackageMapClient.cpp

```diff
@@ -406,6 +406,57 @@ FSpatialNetGUIDCache::FSpatialNetGUIDCache(USpatialNetDriver* InDriver)
 {
 }
 
+typedef TMap<UObject*, uint32> FSubobjectToOffsetMap;
+
+static FSubobjectToOffsetMap CreateOffsetMapFromActor(AActor* Actor, const FClassInfo& Info)
```

Comment:
Nit: no need to specify static on functions declared and used within the same .cpp file
```suggestion
FSubobjectToOffsetMap CreateOffsetMapFromActor(AActor* Actor, const FClassInfo& Info)
```

Reply:
No need, but this prevents this symbol from leaking outside of the cpp and interfering with linking.